Akinmyde/library-backend
# #feature Build out user POST REST api endpoint

__What does this PR do?__
       This PR create a user 

__Description of Task to be completed?__
         Build out POST API endpoint(CRUD)

__How should this be manually tested?__
* Clone this repo https://github.com/Akinmyde/library-backend into any folder of your choice
* cd into the clone folder
* Checkout to this branch by running `npm checkout ft-signup`
* Ensure you have postman on your machine if you don't have use this link to get it installed and running
in your machine https://www.getpostman.com/
* Make a POST request on `/api/v1/auth/signup`
* You will get a response in `JSON format` depending on the inputted value



__What are the relevant pivotal tracker stories?__

__Screenshots__
<img width="1280" alt="Screenshot 2019-03-17 at 11 37 08 PM" src="https://user-images.githubusercontent.com/33469121/54499224-9f739500-490f-11e9-80ef-ebe351debe44.png">
